### ISSUE TITLE
Game Mechanic Issue: Add Stamina Bar

### ISSUE THREAD

**Issue by TommyZynos** [jiminssy/ZhynosAdventure#14]
Seems like there's no way to find out how long your character needs to wait to do the next action cue. It'll be better if we either add a stamina bar or add exhausted Animation. I prefer a stamina Bar.

**Comment by jiminssy** [comment]
Yeah I will have to work with Darryl for this. This was definitely in the plans. We need to put how long the resting is and what are the queued up actions. We need to show this in a very intuitive way to the gamers.

**Comment by jiminssy** [comment]
@dhanslee we need to work on this as top priority after some skill icons are done.
